pkg/ovs/openflow/interfaces.go

```diff
@@ -22,7 +22,6 @@ import (
 )
 
 type Protocol string
-type TableIDType uint8
```

Comment:
Sorry for not get your point, do you mean we should keep using `TableIDType` or just mention that I would remove `TableIDType` and use `uint8` directly in the  commit message? Personally, I feel no need to typed uint8 as `TableIDType`, and we have transform the type in  code like processing packetIn message. @antoninbas

Reply:
I did prefer to have a dedicated type, but I don't feel strongly about it either. If no one else has pointed it out, it's fine to keep it as it is.

Reply:
May I know why you think it is better to use a dedicated type?

Reply:
Because the Antrea code doesn't care about the exact representation of an OpenFlow table ID. It could be 32 bits, 8 bits, a string, as long as it is converted correctly before calling into ofnet. But that's also a matter of preference, and it is fine as it is.